Attempt to fix build error `unknown type name 'uint64_t'`
observed on macOS 10.8 and earlier.

Upstream-Status: Pending
--- server/storage/perfschema/my_thread.h.orig
+++ server/storage/perfschema/my_thread.h
@@ -13,6 +13,8 @@
 #ifdef HAVE_PTHREAD_GETTHREADID_NP
 #include <pthread_np.h>
 #endif
+
+#include <stdint.h>
 
 typedef pthread_key_t thread_local_key_t;
 typedef pthread_t my_thread_handle;
